### DISCUSSION TITLE
Simulating separate populations from same founders and then combining separated populations

### DISCUSSION THREAD

**Discussion by Nfumia** [gaynorr/AlphaSimR#149] [Q&A]
Hello,

First, thank you for all your work with the wonderful package! It has been a great tool to me.

I have a question regarding the simulation of two separate breeding populations and combining them at a set time. Meaning, founders are same, traits are same, all SP same and they are simulated separate for a set number of cycles. I would like to know if there is a way to combine these populations at some point. I apologize in advance if I missed something in the vignettes.

General Scheme (example):
Founders --> POP (cycle 1-10) --[splitPOP]--> POP1(cycle 11-20) & POP2(cycle 11-20) --[combinePOPs???]--> POP1&2(cycle21-30) 

Thank you for your help!

-Nathan

**Comment by gregorgorjanc** [comment]
@Nfumia yes, you can simply combine your population objects after some round of previous separate step, if that’s what you aim for, using `c(popA, popB)`.

**Comment by Nfumia** [comment]
@gregorgorjanc Thank you for your quick response. I had tried that before and it created a list. However, I had not subsetted them so when subsetting the populations to combine the final generations of separation, it works as you recommend. 

For reference: c(POP1,POP2), where POP1 and POP2 are S4 objects with 20 generations each, creates a single S4 object of 40 (lists) generations with the first 20 being POP1 and the second 20 being POP2. But, c(POP1[[20]],POP2[[20]]) creates a single S4 object of 1 (list) generation, combining all slots amended with relevant information.

Thank you again!
